### PR TITLE
Merge 2.8.x into main

### PR DESCRIPTION
**Pull request by rasabot**
:auto_rickshaw: This PR should be merged automatically once it has been approved. If it doesn't happen:
- [ ] Handle merge conflicts
- [ ] Fix build errors

:bulb: It has been opened automatically after changes were merged in a feature branch.

### CODE REVIEW

**Review by m-vdb — APPROVED** [review]

File: changelog/10146.misc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+A new additional slack options `conversation_granularity` is now available  that allows to choose whether interactions across all channels ("sender"), within a channel ("channel") or only within a thread ("thread") will be treated as a single conversation.
```
Comment (carlad) [inline changelog/10146.misc.md:1]: Should "options" be singular, i.e. "option"?

[the remark this answers is not in the document]
Comment (carlad) [reply changelog/10146.misc.md:1]: I created a PR for this: https://github.com/RasaHQ/rasa/pull/10172

[the remark this answers is not in the document]
Comment (m-vdb) [reply changelog/10146.misc.md:1]: @carlad the thing is that this changelog entry has already been reviewed on 2.8.x. If you want to make changes, you'll need to first do them against the 2.8.x branch and then main. Also I think it's from a community member, so I'd be careful not to change things too much unless we have good reasons too 😄

[the remark this answers is not in the document]
Comment (carlad) [reply changelog/10146.misc.md:1]: @m-vdb Ah, yes, apologies I didn't realise. These are only very minor changes so am happy to close that other PR.

[the remark this answers is not in the document]
Comment (m-vdb) [reply changelog/10146.misc.md:1]: no worries, no need to apologise! It's also a good incentive for updating the description of those PRs, to explain what's expected from reviewers more clearly

**Pull request merged**